### ISSUE TITLE
🆕 Software Suggestion | Lemmy

### ISSUE THREAD

**Issue by LongJohn-Silver** [privacytools/privacytools.io#2140]
## Basic Information

**Name:** Lemmy
**Category:** Social Networks/Social News Aggregators
**URL:** https://lemmy.ml/ https://github.com/LemmyNet/lemmy

## Description

Lemmy is similar to sites like Reddit, Lobste.rs, or Hacker News: you subscribe to forums you're interested in, post links and discussions, then vote, and comment on them. Behind the scenes, it is very different; anyone can easily run a server, and all these servers are federated (think email), and connected to the same universe, called the Fediverse.

For a link aggregator, this means a user registered on one server can subscribe to forums on any other server, and can have discussions with users registered elsewhere.

The overall goal is to create an easily self-hostable, decentralized alternative to Reddit and other link aggregators, outside of their corporate control and meddling.

Each Lemmy server can set its own moderation policy; appointing site-wide admins, and community moderators to keep out the trolls, and foster a healthy, non-toxic environment where all can feel comfortable contributing.

## Why I am making the suggestion

Recently Lemmy was able to release its federation so now different instances can be created and they can interact with each other, besides this, it is a libre and privacy respecting alternative to websites like Reddit or Lobsters while also aiming at not providing spaces for right wing organizations. Listing them would also help them to gain some recognition and with more people aware of the project they would probably get more people to help them with development.

As a side note, I think it would be a really awesome idea if the team behind Privacy Tools would add some official moderation in the !privacytools community in the official instance, and/or maybe even hosting an instance. This last part is up to discussion, though.

## My connection with the software

I am a user of the software and administrator in couple communities over there.

- [x] I will keep the issue up-to-date if something I have said changes or I remember a connection with the software.

**Comment by Mikaela** [comment]
It's currently in worth mentioning at https://privacytools.io/providers/social-news-aggregator/

**Comment by LongJohn-Silver** [comment]
@Mikaela Well, still, since now it has reached federation I think it deserves to be listed.

**Comment by atomGit** [comment]
> ... while also aiming at not providing spaces for right wing organizations.

and what about the left-wing organizations?

**Comment by wylel** [comment]
> while also aiming at not providing spaces for right wing organizations.

I want to point out that this is added by the requestor and not in any documentation from Lemmy at all. It would be nice if political views can be kept out of software recommendations.

Lemmy aims to do the same thing Mastodon has done, so it seems, which is fantastic.

**Comment by LongJohn-Silver** [comment]
> > while also aiming at not providing spaces for right wing organizations.
> 
> I want to point out that this is added by the requestor and not in any documentation from Lemmy at all. It would be nice if political views can be kept out of software recommendations.
> 
> Lemmy aims to do the same thing Mastodon has done, so it seems, which is fantastic.

The Lemmy developers are communists and their official instance is pretty strict about not allowing nazis, they also won't help anyone with self hosting and they will block contact with its instance if they found out to be rightists so even though maybe the documentations doesn't say that I know they're into it. And I am totally against not being aware of the political implications of the software you chose, I'm not saying we start a discussion about what branch of communism is better, but stuff like keeping alt-righters away is really important nowadays and I think it is within the scope of Privacy Tools admins to help with that, even more so with recent events.

**Comment by atomGit** [comment]
> It would be nice if political views can be kept out of software recommendations.

those political views were not conjured up in discussion here - they are an intrinsic part of Lemmy and merely pointed out here - at it's core, Lemmy is obviously not centered around freedom

[6. Goals](https://lemmy.ml/docs/lemmy_council.html#6-goals)

* We encourage the membership of groups such as LGBT, religious or ethnic minorities, abuse victims, etc etc, and strive to create a safe space for them to express their opinions. We also support measures to increase participation by the previously mentioned groups.
* The following are banned, and will always be harshly punished: fascism, abuse, racism, sexism, etc etc,

that said, apparently the core values need not be adhered to by 3rd party instances

**Comment by wylel** [comment]
We are talking about 2 different things here. The recommendation is the software, not the home instance.

I am unaware of any place on PTIO that recommends software for the sake of being a fascist, abuser, racist, sexist, etc etc. If the software itself required a personality test before being able to download it and use it, it would not be free software (and as far as I am aware the software licenses do not have this clause in them).

It is fine if you want to back something because of their views on people, politics, or whatever it is, but if it does not play a part in the recommendation of the software itself, and its ability to obtain and use it (in this case your own instance and site, even though it might be disconnected from the rest because of its content) I do not believe it should be a reason to be considered or not.

**Comment by jonaharagon** [comment]
Neat, TIL their main instance is federating now. I agree this should probably be listed.

> and what about the left-wing organizations?

@atomGit bit of a false equivalence I think, seeing as _unlike_ the alt-right, the left is not promoting racism, homophobia, fascism, anti-intellectualism, etc. 🤷‍♂️

**Comment by LongJohn-Silver** [comment]
> Neat, TIL their main instance is federating now. I agree this should probably be listed.

Yes, as far as I know all the instances (except the ones that are hosted by alt-righters) are working together now, the only one I think has some issues is one called Chapo something and I think that's related to them changing various stuff in the code before federation was enabled.

@jonaharagon do you think there's a chance of an official Privacy Tools community on Lemmy? I know it would be hard since there's the forum and Reddit but I feel like it's a good idea.

----

@wylel A lot of times the recommendation is both the software and the main instance, even if not directly, most people will end up using the default instance, but I understand. Still, I am not in the mood to discuss things I've said plenty of times already but I'm just fucking tired of listening to people we should foster spaces for right wing organisations as if they were the same as left wing ones. Please let us end this conversation here, and let it be only about technical or practical reasons why Lemmy should be listed.

**Comment by jonaharagon** [comment]
@LongJohn-Silver I am no longer working with PrivacyTools for various reasons, so I couldn't say. I highly doubt it in the short-/mid-term at least, as I don't believe the current admin team has any intention to host anything new.

**Comment by LongJohn-Silver** [comment]
> @LongJohn-Silver I am no longer working with PrivacyTools for various reasons, so I couldn't say. I highly doubt it in the short-/mid-term at least, as I don't believe the current admin team has any intention to host anything new.

I didn't mean to host an instance, but rather to create a community in the official instance, but I understand. Thanks!

**Comment by jonaharagon** [comment]
IMHO I don't think it would be more beneficial to create a Lemmy community (on lemmy.ml) over creating a Reddit community in the long-run, and in fact because PTIO already has a successful Reddit community it would actually be _more_ harmful to create that kind of fragmentation without significant gain.

To me, the clear benefit to ActivityPub federation and other federations like Matrix is the independence that self-hosting your community grants you, and giving that up to create a community on the flagship instance just to have one doesn't make much sense to me.

**Comment by freddy-m** [comment]
> I didn't mean to host an instance, but rather to create a community in the official instance, but I understand. Thanks!

@LongJohn-Silver we have no intentions of creating an official Lemmy community at this point in time.

**Comment by lrq3000** [comment]
I agree Lemmy should be promoted, it's a stable and functionally exhaustive decentralized alternative to Reddit.
